faucet.config.js

```diff
@@ -3,18 +3,28 @@ let path = require("path");
 
 module.exports = {
 	styleguide: [{
-		source: "./pages.json",
+		source: "./pages.js",
```

Comment:
We discussed that this should be `source: require("./pages.js")` instead. Will implement that tomorrow.

Reply:
Done.